### PR TITLE
[AWS Orgs] Add CloudFormation for IAM roles

### PR DESCRIPTION
**Pull request by orestisfl**
### Summary of your changes

Adds the CloudFormation template to create IAM roles in the management and member accounts

### Screenshot/Data
image
image
image

### Related Issues

- Related: #982
- Fixes: #1036 

### Checklist
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary README/documentation (if appropriate)

### CODE REVIEW

File: deploy/cloudformation/organization.yml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Creates IAM roles needed for multi-account access
+Parameters:
+  OrganizationalUnitIds:
+    Type: CommaDelimitedList
+    Description: |
+      Comma-separated list of organizational units to deploy the IAM roles to.
+      Specify the unique IDs of the organizational units where the roles should be deployed.
+      Example: ou-abc123,ou-def456,ou-ghi789
+    AllowedPattern: '.+'
```
Comment (amirbenun) [inline deploy/cloudformation/organization.yml:11]: That's really cool, how is it populated?

**Review by amirbenun — COMMENTED** [review]
Nice!

File: deploy/cloudformation/organization.yml (hunk 1 above)
Comment (orestisfl) [reply deploy/cloudformation/organization.yml:11]: Essentially the first screenshot. The ids come from the AWS organizations page.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -59,16 +59,16 @@ repos:
...
         args: [ -i ]
-        files: deploy/cloudformation/elastic-agent-ec2.yml
+        files: deploy/cloudformation/.*yml
 
   - repo: https://github.com/awslabs/cfn-python-lint
     rev: v0.77.5
     hooks:
     -   id: cfn-python-lint
-        files: deploy/cloudformation/elastic-agent-ec2.yml
+        files: deploy/cloudformation/(organization|elastic-agent-ec2).yml
```
[the remark this answers is not in the document]
Comment (orestisfl) [reply .pre-commit-config.yaml:71]:
```
AWS CloudFormation Linter................................................Failed
- hook id: cfn-python-lint
- exit code: 4

W8001 Condition Encrypted not used
deploy/cloudformation/ec2-types.yml:16:3
```

**Review by amirbenun — APPROVED** [review]

**Pull request merged**